telemetry/definitions/commonDefinitions.json

```diff
@@ -3145,6 +3190,80 @@
                 { "type": "codecatalyst_devEnvironmentWorkflowStep"},
                 { "type": "codecatalyst_devEnvironmentWorkflowError", "required": false }
             ]
+        },
+        {
+            "name": "codewhispererterminal_userLoggedIn",
+            "description": "Emitted when users log in",
+            "passive": false,
+            "metadata": []
+        },
+        {
+            "name": "codewhispererterminal_completionInserted",
+            "description": "Emitted on Autocomplete user decisions",
+            "passive": false,
+            "metadata": [
```

Comment:
are all `codewhispererterminal_completionInserted`  `passive:false` ?

Reply:
will this metric be used in calculating user success rate? if yes then consider adding result field.